app/controllers/users_controller.rb

```diff
@@ -1,10 +1,12 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!
   def index
     @users = User.order(:id)
   end
 
   def show
-    @show_user = User.find(params[:id])
+    # @show_user = User.find(params[:id])
```

Comment:
- [ ] Please remove any unnecessary code for a cleaner appearance and better optimization. 👍